BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceViewController.swift

```diff
@@ -95,9 +95,8 @@ final class DigitalInvoiceViewController: UIViewController {
     }
 
     private func setupView() {
-        title = NSLocalizedStringPreferredFormat("ginibank.digitalinvoice.screentitle",
-                                                 comment: "Digital invoice")
-
+        title = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.screentitle",
+                                                    comment: "Digital invoice")
```

Comment:
[Suggestion] a indentation would be nice here, to align `comment` parameter in the right place

Reply:
Good observation, thank you!